bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -104,6 +105,8 @@
     protected static final String GC_OVERREPLICATED_LEDGER_WAIT_TIME = "gcOverreplicatedLedgerWaitTime";
     protected static final String USE_TRANSACTIONAL_COMPACTION = "useTransactionalCompaction";
     protected static final String VERIFY_METADATA_ON_GC = "verifyMetadataOnGC";
+    protected static final String GC_PERSISTENT_ENTRYLOGMETADATA_MAP_ENABLED = "gcPersistentEntrylogMetadataMapEnabled";
```

Comment:
I think a better name for these two settings would be:

`entryLogMetadataCacheEnabled` and `entryLogMetadataCachePath`

Reply:
@sijie sorry for late response. I think I missed this notification. I addressed all the comments.